MapleServer2/PacketHandlers/Game/ShopHandler.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
```

Comment:
Remove unnecessary import. Also there is a bunch of formatting issues, simply run `dotnet-format` to fix those.